src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+type Props = {
+  value: number;
+  setValue: React.Dispatch<React.SetStateAction<number>>;
+  setPage: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export const DropDown: React.FC<Props> = ({ value, setValue, setPage }) => {
+  const handleSelectChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    event.preventDefault();
+    setValue(Number(event.target.value));
+    setPage(1);
```

Comment:
Setting the page to 1 directly inside the `handleSelectChange` function makes this component less reusable. Consider calling a prop function that can handle this logic outside of the component, making the component more flexible.